site/content/en/docs/manual/basics/attach-cloud-storage.md

```diff
@@ -0,0 +1,179 @@
+---
+title: 'Attach cloud storage'
+linkTitle: 'Attach cloud storage'
+weight: 21
+description: 'Instructions on how to attach cloud storage using UI'
+---
+
+In CVAT you can use AWS-S3 and Azure Blob Container cloud storages to store image datasets for your tasks.
+
+## Using AWS-S3
+
+### Create AWS account
+
+First, you need to create an AWS account, to do this, [register of 5 steps](https://portal.aws.amazon.com/billing/signup#/start)
+following the instructions
+(even if you plan to use a free basic account you may need to link a credit card to verify your identity).
+
+To learn more about the operation and benefits of AWS cloud,
+take a free [AWS Cloud Practitioner Essentials](https://www.aws.training/Details/eLearning?id=60697) course,
+which will be available after registration.
+
+### Create a bucket
+
+After the account is created, go to [console AWS-S3](https://s3.console.aws.amazon.com/s3/home)
+and click `Create bucket`.
+
+![](/images/aws-s3_tutorial_1.jpg)
+
+You'll be taken to the bucket creation page. Here you have to specify the name of the bucket, region,
+optionally you can copy the settings of another bucket by clicking on the `choose bucket` button.
+Checkbox block all public access can be enabled as we will use `access key ID` and `secret access key` to gain access.
+In the following sections you can leave the default settings and click `create bucket`.
+After you create the bucket it will appear in the list of buckets.
+
+### Create user and configure permissions
+
+To access bucket you will need to create a user, to do this, go [IAM](https://console.aws.amazon.com/iamv2/home#/users)
+and click `add users`. You need to choose AWS access type have access key ID and secret access key.
+
+![](/images/aws-s3_tutorial_2.jpg)
+
+After pressing `next` button to configure permissions, you need to create a user group.
+To do this click `create a group`, input the `group name` and select permission policies add `AmazonS3ReadOnlyAccess`
+using the search (if you want the user you create to have write rights to bucket select `AmazonS3FullAccess`).
+
+![](/images/aws-s3_tutorial_3.jpg)
+
+You can also add tags for the user (optional), and look again at the entered data. In the last step of creating a user,
+you will be provided with `access key ID` and `secret access key`,
+they will need to be used in CVAT when adding cloud storage.
+
+![](/images/aws-s3_tutorial_4.jpg)
+
+### Upload dataset
+
+Initially you need to create a manifest file for your image dataset. Information on how to do that is available
+on the [Simple command line to prepare dataset manifest file](/docs/manual/advanced/dataset_manifest) page.
```

Comment:
@TOsmanov , let's repeat here basic steps to upload a simple dataset. Need exact instructions. Let's take a video file (take a link from serverless tutorial), prepare a manifest and upload them.

Reply:
@nmanovic , as far as I understand cloud storage into UI does not support the use of video file, do you want me to describe the division of the video into frame sequence?

Reply:
@Marishka17 , could you please comment on that?

Reply:
@TOsmanov , OK. Let's use a dataset with images in the case.